docs/development.md

```diff
@@ -0,0 +1,18 @@
+# Development Notes
+
+## Arrow Functions
+This projet heavily prefers the use of the `function` keyword over the use of arrow functions in most cases.
+Arrow functions may be used in simple callbacks in functional functions such as `map` or `filter` and should not be more than one line long.
```

Comment:
This part is a bit ambiguous:
"simple callbacks in functional functions".
What about a non-trivial callback, or a callback that is used in a non-trivial way?
What about "simple callbacks" that are not functional?

You may want to re-conceptualize this a bit.

The one line long part is good.